pkg/analytics/models.go

```diff
@@ -0,0 +1,140 @@
+package analytics
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"time"
+
+	otellog "go.opentelemetry.io/otel/log"
+
+	"github.com/bacalhau-project/bacalhau/pkg/models"
+)
+
+type EventType string
+
+const (
+	JobEventType = "job"
```

Comment:
lets add versioning to event types so that change in pubished schema wouldn't break things

Reply:
done, good idea.